README.asciidoc

```diff
@@ -369,6 +369,19 @@ command names:
 ------------------------------------------------------------------------------------------
 
 
+[[Scripting]]
```

Comment:
`README.asciidoc` has a manual TOC — happy to learn about a better way that works! — so `Scripting` will not appear in the TOC… unless you add an entry to it to match.
Or move it to known limitations? (Unsure about that.)

Reply:
I'm more of a markdown fan, never used asciidoc before…so I don't know, sorry